pypuf/learner/evolution_strategies/becker.py

```diff
@@ -0,0 +1,311 @@
+import numpy as np
+import scipy.stats as stats
+import itertools
+from pypuf import tools
+from pypuf.simulation.arbiter_based.ltfarray import LTFArray
+
+class Reliability_based_CMA_ES():
+
+    # recommended properties of parameters:
+    #   pop_size=30
+    #   parent_size=10 (>= 0.3*pop_size)
+    #   priorities: linear low decreasing
+    def __init__(self, instance, pop_size, parent_size, priorities, challenge_num, repeat, unreliability):
+        self.transform = LTFArray.transform_id
+        self.combiner = LTFArray.combiner_xor
```

Comment:
This doesn't really need to be a variable -- I do not believe this algorithm would work if we do not use XOR